lxd/network/driver_ovn.go

```diff
@@ -2553,9 +2553,22 @@ func (n *ovn) start() error {
 
 	var err error
 	var projectID int64
+	var node db.NodeInfo
 	err = n.state.Cluster.Transaction(func(tx *db.ClusterTx) error {
+		// Get the project ID.
 		projectID, err = tx.GetProjectID(n.project)
-		return err
+		if err != nil {
+			return err
+		}
+
+		// Get the node info.
+		nodeID := n.state.Cluster.GetNodeID()
```

Comment:
This is just a nit, but as you're inside a transaction, it seems cleaner to use tx.GetNodeID() (although i'm aware its makes no actual difference to the data access).